etc/afpd/directory.c

```diff
@@ -1216,14 +1216,14 @@ struct path *cname(struct vol *vol, struct dir *dir, char **cpath)
                  * this will terminate clean in while (1) because len == 0,
                  * probably afp_createfile|dir
                  */
-                LOG(log_maxdebug, logtype_afpd, "came('%s'): {leave-cnode ENOENT (possile create request): '%s'}",
+                LOG(log_maxdebug, logtype_afpd, "cname('%s'): {leave-cnode ENOENT (possile create request): '%s'}",
```

Comment:
For the record: There's actually one more typo on this line "possile create request" which I assume is supposed to say "possible".... which is still awkward English. Maybe the author meant to say "possible to create request"?

Reply:
Yes, you are right, there are a lot of typos in the code, not only in this place. I fixed only the one that seemed important to me for understanding debug output. Honestly, I even did not notice that the word "possible" was with a typo too :).

Reply:
I even found a Debian-specific patch in their distro, exclusively related to typos: https://sources.debian.org/patches/netatalk/3.1.14~ds-1/103_fix_typos.patch/
I was surprised that they started patching the source inside the distro instead of just committing it to upstream.

Reply:
Wow, good job tracking down that patch! I made a PR for upstream (while making a few more typo fixes that I have noticed) -- https://github.com/Netatalk/Netatalk/pull/218